src/components/HtmlView.js

```diff
@@ -73,23 +69,30 @@ export const HtmlView = memo(({ html, tagsStyles, openWebScreen, width }) => {
 
   return (
     <HTML
-      html={html}
+      source={{ html }}
       {...htmlConfig}
-      onLinkPress={(evt, href) => openLink(href, openWebScreen)}
+      renderersProps={{
+        a: { onPress: (evt, href) => openLink(href, openWebScreen) },
+        table: { cssRules }
+      }}
       tagsStyles={{ ...styles.html, ...tagsStyles }}
       emSize={normalize(16)}
-      baseFontStyle={styles.baseFontStyle}
+      baseStyle={{ ...styles.baseFontStyle }}
```

Comment:
why are you destructing here? what about `baseStyle={styles.baseFontStyle}`

Reply:
text in the [document](https://meliorence.github.io/react-native-render-html/api/stylesconfig#basestyle):

- Style for the root element. Inheritable properties will be transferred recursively to child nodes, including text styles.

Reply:
we spoke on slack and cleared up the misunderstanding. code has been edited.

https://github.com/smart-village-solutions/smart-village-app-app/commit/bcc80d212f5cbe719c85ba9c5c36b2df18629b96